monitoring/api/v3/cloud-client/snippets.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+from google.cloud.monitoring import (
+    Aligner, Client, MetricKind, Reducer, ValueType)
+
+
+def create_metric_descriptor(client):
+    descriptor = client.metric_descriptor(
+        'custom.googleapis.com/my_metric',
+        metric_kind=MetricKind.GAUGE,
+        value_type=ValueType.DOUBLE,
+        description='This is a simple example of a custom metric.')
+    descriptor.create()
+
+
+def delete_metric_descriptor(client, descriptor):
+    descriptor = client.metric_descriptor(
+        'custom.googleapis.com/my_metric'
+    )
+    descriptor.delete()
+    print 'Deleted metric.'
+
+
+def write_time_series(client):
+    resource = client.resource(
+        'gce_instance',
+        labels={
+            'instance_id': '1234567890123456789',
+            'zone': 'us-central1-f',
+        }
+    )
+
+    metric = client.metric(
+        type_='custom.googleapis.com/my_metric',
+        labels={
+            'status': 'successful',
+        }
+    )
+    client.write_point(metric, resource, 3.14)
+
+
+def list_time_series(client):
+    metric = 'compute.googleapis.com/instance/cpu/utilization'
+    print(list(client.query(metric, minutes=5)))
```

Comment:
prefer a temporary variable with printing things out, and in general, it's better for a list/iterator to be shown using a for loop:

```python
results = client.query(metric, minutes=5)

for result in results:
    print(result)
```